language_detector/main.py

```diff
@@ -1,8 +1,9 @@
-# -*- coding: utf-8 -*-
-
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counts = { lang['name']:0 for lang in languages }
+    words = text.split()
+    for w in words:
```

Comment:
@jsymons thanks for the review!

agree with `for word` - it is far more descriptive. however, I think `words = text.split()` is good because it makes it explicit that we're splitting the string by spaces into a collection of words.

code is written once and read multiple times, after all. unless one is explicitly going for code golf, etc, imho sometimes it's better to have an extra line if it means it's clearer for whoever works on the code in the future (and it can be the same person who wrote the code). nothing like wondering /why/ you made the decisions you did.